python/lsst/cp/pipe/ptc/cpSolvePtcTask.py

```diff
@@ -500,7 +546,8 @@ def fitDataFullCovariance(self, dataset):
         have r^2 fewer entries.
         """
         matrixSide = self.config.maximumRangeCovariancesAstier
-        lenParams = matrixSide*matrixSide
+        matrixSideFit = self.config.maximumRangeCovariancesAstierFullCovFit
```

Comment:
Should this parameter be pulled from the dataset itself?  It's already been set during initialization, right?